### ISSUE TITLE
Bag Search

### ISSUE THREAD

**Issue by nhens0** [Jaliborc/Bagnon#1138]
When searching in bag the name of the inventory blocks you from being able to see what you are typing. Is there a way to remove that so it doesn't show Character name Inventory off of mine or completely I think that you know that it is your inventory. Sometimes it bugs where the letters are together. 
![bag](https://user-images.githubusercontent.com/59795441/72219200-ff73a180-3508-11ea-8316-c5cc2fdd0cd0.png)

**Comment by Doshilol** [comment]
Same issue here. It also makes it harder to type something in the search again or clear it out, because it seems the "idle" search-free UI header with "Character_name's inventory" is active at the same time, so double-clicking doesn't always work. The latest 8.3.3 update, unfortunately, didn't help.

**Comment by Jaliborc** [comment]
Do you want the title to hide when the search box is open? I'm not sure if I understand what you are asking.

**Comment by Doshilol** [comment]
![photo_2020-01-30_22-23-09](https://user-images.githubusercontent.com/44231059/73462937-1b06f500-43af-11ea-865c-8d633d216f0c.jpg)
No, the issue is when you try to find something, it looks like this. The search works, but there is an overlap of text in the UI, sometimes making it hard to clean this textbox, because the "Charname's Inventory" is pressed instead, being active at the same time.

**Comment by nhens0** [comment]
I would love for an option to had the players name inventory all together. I just dont know what is easier to code :)

**Comment by raysmith59** [comment]
Yeah, I came to report this as well, used to be the "Toonname's Inventory" would disappear when you started typing into that field, doesn't happen anymore.  As well, the "click to search" icon doesn't highlight anymore when a search is happening, probably broke at the same time.
